addon/helpers/choice.js

```diff
@@ -9,6 +9,11 @@ export default MicroState.extend({
       return Type.create(values, options);
     },
 
+    set(choice, values, options) {
```

Comment:
Yep, I get that part, but it makes set impractical for Choice because it requires the user to understand how to build a working Choice object. I'm ok to punt on this if you're ok with `set` remaining in this state for Choice.